v5_position_service.go

```diff
@@ -84,3 +88,37 @@ func (s *V5PositionService) GetPositionInfo(param V5GetPositionInfoParam) (*V5Ge
 
 	return &res, nil
 }
+
+// V5SetLeverageParam :
+type V5SetLeverageParam struct {
+	Category     CategoryV5 `json:"category"`
+	Symbol       *SymbolV5  `json:"symbol"`
+	BuyLeverage  *string    `json:"buyLeverage"`
+	SellLeverage *string    `json:"sellLeverage"`
+}
```

Comment:
<img width="990" alt="スクリーンショット 2023-03-06 22 46 16" src="https://user-images.githubusercontent.com/12891407/223127738-787c5bd1-8131-4668-9d0e-0b38a60ab278.png">

```suggestion
// V5SetLeverageParam :
type V5SetLeverageParam struct {
	Category     CategoryV5 `json:"category"`
	Symbol       SymbolV5  `json:"symbol"`
	BuyLeverage  string    `json:"buyLeverage"`
	SellLeverage string    `json:"sellLeverage"`
}
```

Because all params are required.